### PR TITLE
US-2057 Added original tx data to Redux

### PR DESCRIPTION
**Pull request by Freshenext**

### CODE REVIEW

File: src/screens/send/types.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ export interface TransactionInformation {
   value?: string
   symbol?: string
   hash?: string
+  original?: object
```
Comment (jessgusclark) [inline src/screens/send/types.ts]: This should be `Transaction` from ethers.

File: src/screens/send/types.ts [hunk 2]
```diff
@@ -63,6 +65,7 @@ type TransferTransactionStatusPending = {
 
 type TransferTransactionStatusConfirmed = {
   txStatus: 'CONFIRMED'
+  original?: object
```
Comment (jessgusclark) [inline src/screens/send/types.ts]: Same type here.

**Review by jessgusclark — CHANGES_REQUESTED** [review]
This looks good and works well, just a small type change to help developers.

File: src/screens/send/types.ts (hunk 1 above)
Comment (Freshenext) [reply src/screens/send/types.ts]: Good to go - I've used the TransactionResponseWithoutWait

File: src/screens/send/types.ts (hunk 2 above)
Comment (Freshenext) [reply src/screens/send/types.ts]: Good to go - I've used TransactionResponseWithoutWait

**Review by jessgusclark — APPROVED** [review]

**Pull request merged**